src/pages/LoginPage.js

```diff
@@ -1,16 +1,102 @@
-import React from "react";
-import { Link } from "react-router-dom";
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import "../styles/LoginPage.css";
 
-function LoginPage() {
+export default function Login() {
+  const [formData, setFormData] = React.useState({ email: "", password: "" })
+  const [item, setItems] = React.useState([])
```

Comment:
Naming things is super important. Items doesn't really say what this is. since we are storing a _list of user_ here, name it something appropriate along those lines.

Reply:
Changed the naming convention.